base/math.jl

```diff
@@ -1327,7 +1327,12 @@ end
     x0 = x
     n == 3 && return x*x*x # keep compatibility with literal_pow
     if isodd(n)
-        y = n < 0 ? inv(x) : x
+        if n < 0
+            y = inv(x)
+            ynlo = -fma(x, y, -1.0) * y
```

Comment:
I would also rewrite this to `ynlo = fma(-x, y, 1.0)* y`. I'm not sure why I did it with the double negation previously.